web/src/components/tasks/license-calendar-event/LicenseCalendarEventElement.tsx

```diff
@@ -0,0 +1,87 @@
+import { Content } from "@/components/Content";
+import { HorizontalLine } from "@/components/HorizontalLine";
+import { SingleCtaLink } from "@/components/njwds-extended/cta/SingleCtaLink";
+import { LicenseCurrentStatusComponent } from "@/components/tasks/LicenseCurrentStatusComponent";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import { useUserData } from "@/lib/data-hooks/useUserData";
+import { LicenseEventType } from "@/lib/types/types";
+import { parseDate, parseDateWithFormat } from "@businessnjgovnavigator/shared/dateHelpers";
+import { defaultDateFormat } from "@businessnjgovnavigator/shared/defaultConstants";
+import { LicenseName, licenseSearchDateFormat } from "@businessnjgovnavigator/shared/license";
+import { LicenseEventSubtype } from "@businessnjgovnavigator/shared/taxFiling";
+import { Business } from "@businessnjgovnavigator/shared/userData";
+import { ReactElement } from "react";
+
+interface LicenseElementProps {
+  license: LicenseEventType;
+  licenseEventType: LicenseEventSubtype;
+  licenseName: string;
+  CMS_ONLY_fakeBusiness?: Business;
+  CMS_ONLY_fakeLicenseName?: LicenseName;
+}
+
+export const LicenseElement = (props: LicenseElementProps): ReactElement => {
+  const userDataFromHook = useUserData();
+  const business = props.CMS_ONLY_fakeBusiness ?? userDataFromHook.business;
+  const { Config } = useConfig();
+
+  const licenseName = props.CMS_ONLY_fakeLicenseName ?? props.license.licenseName;
+  const licenseDetails = business?.licenseData?.licenses?.[licenseName];
+  const expirationDateISO = licenseDetails?.expirationDateISO;
+
+  let date = parseDate(expirationDateISO);
+  if (props.licenseEventType === "renewal") {
+    date = date.add(30, "days");
+  }
+
+  return (
+    <div className="minh-38">
+      <div className="bg-base-extra-light margin-x-neg-4 margin-top-neg-4 radius-top-lg">
+        <div className="padding-top-4 padding-bottom-4 margin-x-4">
+          <h1>
+            {props.licenseEventType === "renewal"
+              ? props.license.renewalEventDisplayName
+              : props.license.expirationEventDisplayName}
+          </h1>
+          <div>
+            <span className="text-bold">
+              {" "}
+              {Config.currentLicenseStatusDefaults.licenseExpirationHeaderText}
+            </span>{" "}
```

Comment:
dust - not sure why we have random {" "} stuff going on here, care to clarify for me?

Reply:
I delete the space on line 48, it's not needed, but the {" "} creates a space between the span and the text that immediately follows it.

Reply:
It's another way of adding a space